mlir/test/Conversion/GPUToLLVMSPV/gpu-to-llvm-spv.mlir

```diff
@@ -167,13 +337,13 @@ gpu.module @shuffles attributes {
                           %val2: f32, %delta_up: i32,
                           %val3: f64, %delta_down: i32) {
     %width = arith.constant 16 : i32
-    // CHECK:         llvm.call spir_funccc @_Z17sub_group_shuffleij(%[[VAL_0]], %[[VAL_1]]) : (i32, i32) -> i32
+    // CHECK:         llvm.call spir_funccc @_Z17sub_group_shuffleij(%[[VAL_0]], %[[VAL_1]]) {{{.*}}} : (i32, i32) -> i32
```

Comment:
I would probably just strip the return type, as this is either needs match the signature of the function, which is verified.

```suggestion
    // CHECK:         llvm.call spir_funccc @_Z17sub_group_shuffleij(%[[VAL_0]], %[[VAL_1]])
```

Reply:
Thanks, I've done that here: https://github.com/llvm/llvm-project/pull/99301/commits/4a11de02c8c4860268f772512521a10290a5be05